web/app/controllers/drivingSchoolCtrl.js

```diff
@@ -0,0 +1,42 @@
+var DrivingSchool = require("../models/DrivingSchool");
+
+exports.createSchool = function(req, res) {
```

Comment:
You are repeating behavior that could be extracted out like so
```
exports.createSchool = function(req, res) {
    var requiredItems = ["addressLine1", "state", "zip"];
    var missingItems = GetMissingInformation(req.body, requiredItems);

    if (missingItems.length > 0) {
        res.status(400)
        res.json({
            "missing-items": missingItems
        });
        return;
    }
    var newSchool = new DrivingSchool({
        addressLine1: req.body.addressLine1,
        addressLine2: req.body.addressLine2,
        state: req.body.state,
        zip: req.body.zip
    });
    newSchool.save(function(err) {
        if (err) {
            res.status(400);
            res.json({"message":"Error creating a new school", "error": err});
        } else {
            res.status(201);
            res.json(newSchool);
        }
    });
}

function GetMissingInformation(jsonObject, keys) {
    var missingKeys = [];
    for (var key in keys) {
        if (!jsonObject.hasOwnProperty(keys[key]))
            missingKeys.push(keys[key]);
    }
    return missingKeys
}
```

Also, I would recommend always just responding with json, considering this is an api.